index.php

```diff
@@ -24,6 +24,26 @@
     exit();
 }
 
+/**
+ * HTTP Security Headers
+ */
+// HTTP Strict Transport Security (HSTS) - two years
+header("Strict-Transport-Security: max-age=63072000");
+// Content Security Policy (CSP) - starting with recommended locked down policy
+header("Content-Security-Policy: default-src 'none';
+	img-src 'self';
+	script-src 'self' ajax.googleapis.com use.fontawesome.com;
+	style-src 'self' fonts.googleapis.com ajax.googleapis.com use.fontawesome.com;
```

Comment:
Best way to figure out if we've got everything we need, is to look at page source of a standard page? Just wondering about how to make sure we're not missing something that would break stuff. This list looks right to me off the top of my head, but would be nice if we can verify somehow.